receiver/otlpreceiver/go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	github.com/rs/cors v1.9.0 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.opentelemetry.io/collector/exporter v0.79.0 // indirect
+	go.opentelemetry.io/collector/extension v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
Should this be `v0.79.0` with a replace?

Reply:
i'll update in a following PR to pull in what's in main, this will make life easier for update core in contrib